crates/pil-air-builder/src/lib.rs

```diff
@@ -0,0 +1,159 @@
+mod symbolic_builder;
```

Comment:
Since we can export the columns via `BaseAirWithPublicValues`, having this here is not really necessary, so in case the other change is OK, the whole `pil-air-builder` should disappear.